spp_change_request/views/registrant_view.xml

```diff
@@ -8,6 +8,10 @@ Part of OpenSPP. See LICENSE file for full copyright and licensing details.
         <field name="name">view_group_ext_form_custom_spp_cr</field>
         <field name="model">res.partner</field>
         <field name="inherit_id" ref="g2p_registry_group.view_groups_form" />
+        <field
+            name="groups_id"
+            eval="[(4, ref('g2p_registry_base.group_g2p_registrar')), (4, ref('g2p_registry_base.group_g2p_admin'))]"
+        />
         <field name="arch" type="xml">
             <xpath expr="//page[@name='other']" position="before">
                 <page string="Change Requests">
```

Comment:
```suggestion
        <field name="arch" type="xml">
            <xpath expr="//page[@name='other']" position="before">
                <page string="Change Requests" groups="g2p_registry_base.group_g2p_registrar,g2p_registry_base.group_g2p_admin">
```

Reply:
@nhatnm0612 Already tried this but didn't fixed the error since it still renders the field below <page> element which causes the error.

Reply:
if you want to hide fields before it's rendered to the views, there is a param inside fields called groups which will do the same thing.

groups ([str](https://docs.python.org/3/library/stdtypes.html#str)) – comma-separated list of group xml ids (string); this restricts the field access to the users of the given groups only